users/csc027/csc027.c

```diff
@@ -0,0 +1,63 @@
+#include "csc027.h"
+
+static const char* git_macros[] = {
```

Comment:
Also, just thinking about it, you could probably set this as PROGMEM, so it's not loaded into memory.   Though, you'd need to use `send_string_P`, below, too.

Reply:
I'm not too familiar with AVR, but from what I can tell, I would need to declare the individual strings separately to store them in memory.  For example, declare the strings as
```c
static const char macro_0[] PROGMEM = "git add ";
static const char macro_1[] PROGMEM = "git branch ";
...
```
, then point to them in
```c
static const char* const git_macros[] = {
    macro_0,
    macro_1,
    ...
}
```
I've also tried storing the array in `PROGMEM` via
```c
static const char* const git_macros[] PROGMEM = {
    macro_0,
    macro_1,
    ...
}
```
,but it requires a change to the accessor:
```c
send_string_P((char*)pgm_read_word(&git_macros[keycode - GIT_ADD]));
```
Is there some way to keep the strings in the array initializer, while keeping the strings in `PROGMEM`?  If not, I think storing the strings in memory might be acceptable, if only for the readability benefits.  I've tried other combinations, but they either garble output or aren't exactly the semantics I want (i.e., strings stored in `PROGMEM` and array stored in `PROGMEM`).

Reply:
I believe that you can use them as a progmem, but I'd have to double check.

I think for now, it should be fine to use it as is.

Reply:
I'll keep it as is then.  If I find something down the line that cleans up the declaration, I'll open another PR.  Also, thanks for reviewing this one.  :)